Extension/src/LanguageServer/configurations.ts

```diff
@@ -53,7 +53,7 @@ export interface ConfigurationJson {
 }
 
 export interface Configuration {
-    name: string;
+    name?: string;
```

Comment:
This field is required.  We should figure out where in the code is not expecting a name and fix it.